src/document/UIData.tsx

```diff
@@ -148,6 +149,12 @@ export const ViewData = {
     name: "Focus",
     icon: <CropFree />,
     default: false
+  },
+  FieldOverlays: {
+    index: 6,
+    name: "FieldOverlays",
```

Comment:
This naming is too vague and inconsistent with internal naming as well (everything is considered an Overlay).